internal/pkg/deploy/cloudformation/cc_pipeline_integration_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integration
+// +build integration,deploy
```

Comment:
nit: I like the idea! can we instead change the tag to `integration-remote` to be more specific? I feel like it's possible for us to use the `deploy` build tag for possibly another scenario

Reply:
Good idea! Done.